cmd/minishift/cmd/start.go

```diff
@@ -243,6 +244,30 @@ func applyAddOns(ip string, routingSuffix string, ocPath string, kubeConfigPath
 	}
 }
 
+// TODO - persistent volume creation should really be fixed upstream, aka 'cluster up' (HF)
+// configurePeristentVolumes makes sure that the default persistent volumes created by 'cluster up' have the right permissions - see https://github.com/minishift/minishift/issues/856
+func configurePeristentVolumes(sshCommander provision.SSHCommander) {
+	fmt.Println("-- Waiting for persistent volumes to be ready")
+
+	hostPvDir := viper.GetString(hostPvDir)
+
+	// there are in total 101 directories when using ls. There is an additonal 'registry' directory
+	cmd := fmt.Sprintf("while [ $(ls %s | wc -l) -lt 101 ] ; do sleep 1; done", hostPvDir)
```

Comment:
BTW, it seems when 'cluster up' is used on a native Docker daemon, the problem does not exist. It seems to be an issue related to the fact that we are running in a VM. So I would feel a bit better, if I'd understand why that is.

@csrwng, any idea?